in_progress/mission-statement.md

```diff
@@ -0,0 +1,53 @@
+# Mission Statement
+
+## Goals
+
+We want to make single cell data analysis as accesible as possible. This means
+
+* Keeping data open, prioritizing interoperable data formats over serialized objects
+* Diversity, we believe in removing barriers to participation in research, including economic factors and institutional culture. Our tools will have the most impact if they are developed by and for a diverse community.
+* Transparency, we do our work as openly as possible and welcome our community to join in the decision making process
+* Simplicity – Invent as little as possible and work with standard pydata types. Our users already have to be experts in multiple-fields. Lets not raise the bar for usage higher.
+
+## Abstract
+
+Scanpy and accessory packages for single-cell data analysis in Python have seen a strong and steady increase in interest since its first publication.
+The ecosystem has outgrown a single package and modality, so we've formed a new organization, the **`scverse`**, to handle this.
+
+The `scverse` is a community-based ecosystem of tools with users and developers across many organizations.
+For the ecosystem to be useful and grow, it’s essential that the core analytic tools are well made and continue to be maintained. Frequently, tools are made by small groups within individual labs, and can quickly fall into disrepair once a grad student leaves. To some extent, this is unavoidable and is the nature of academic code. For commonly used, important tools this needs to be avoided.
+
+The scverse constitutes an environment of tools which is deliberately not unified under a single package, but instead a consortium of core analytic tools with shared maintanance resposibilities.
+Tools within this environment that are considered "core" for sharing data between tools or are required for fundamental analyses of single cell datatypes are placed under shared maintainance in the `scverse` github organization.
```

Comment:
```suggestion
A tool within this environment can be considered "core" if it is designed for sharing data between tools or is required for fundamental analyses of single cell datatypes. These tools are placed under shared maintenance in the `scverse` GitHub organization.
```

Corrected grammar, but this sentence is convoluted? Then what's not core?

Reply:
is scverse just the core tools?

Reply:
```suggestion
We define "core tools" as those which facilitate sharing data through common formats, or provide foundational support for single cell datatypes (e.g. modality specific IO, toolkits) and analyses. 
These tools are placed under shared maintenance in the `scverse` GitHub organization.
```